scripts/inventory/consul_io.py

```diff
@@ -324,7 +324,7 @@ def load_data_for_node(self, node, datacenter):
 
         self.load_groups_from_kv(node_data)
         self.load_node_metadata_from_kv(node_data)
-        if self.config.suffixes == 'true':
+        if self.config.bulk_load != 'true':
```

Comment:
Same here.

Reply:
My guess of this is that previously because the upper if statement uses `suffixes` so it uses the same here.

Reply:
This change is not OK because now extra data will not be loaded the required.

Reply:
Hi, @tadeboro 
Sorry for the late reply, I've been busy for the past couple of days.

After some investigation, 
I found out that the script doesn't even required the `if self.config.suffixes == 'true':` statement

Use the same example as this PR.
After removing the if statement
```
def load_data_for_node(self, node, datacenter):
    '''loads the data for a single node adding it to various groups based on
    metadata retrieved from the kv store and service availability'''

    if self.config.bulk_load == 'true':
        node_data = self.consul_get_node_inmemory(node)
    else:
        index, node_data = self.consul_api.catalog.node(node, dc=datacenter)
    node = node_data['Node']

    self.add_node_to_map(self.nodes, 'all', node)
    self.add_metadata(node_data, "consul_datacenter", datacenter)
    self.add_metadata(node_data, "consul_nodename", node['Node'])
    self.load_groups_from_kv(node_data)
    self.load_node_metadata_from_kv(node_data)
    
    self.load_availability_groups(node_data, datacenter)
    for name, service in node_data['Services'].items():
        self.load_data_from_service(name, service, node_data)
```

Result with suffixes enabled
```
➜ ansible-inventory -i consul_io.py --playbook-dir . --graph
@all:
  |--@agent1_servers:
  |  |--172.19.0.4
  |--@agent1_servers_tag1:
  |  |--172.19.0.4
  |--@agent1_servers_test:
  |  |--172.19.0.4
  |--@agent2_servers:
  |  |--172.19.0.2
  |--@agent2_servers_tag2:
  |  |--172.19.0.2
  |--@agent2_servers_test:
  |  |--172.19.0.2
  |--@consul_servers:
  |  |--172.19.0.3
  |--@dc1:
  |  |--172.19.0.2
  |  |--172.19.0.3
  |  |--172.19.0.4
  |--@ungrouped:
```
Result with suffixes disabled
```
➜ ansible-inventory -i consul_io.py --playbook-dir . --graph
@all:
  |--@agent1:
  |  |--172.19.0.4
  |--@agent1_tag1:
  |  |--172.19.0.4
  |--@agent1_test:
  |  |--172.19.0.4
  |--@agent2:
  |  |--172.19.0.2
  |--@agent2_tag2:
  |  |--172.19.0.2
  |--@agent2_test:
  |  |--172.19.0.2
  |--@consul:
  |  |--172.19.0.3
  |--@dc1:
  |  |--172.19.0.2
  |  |--172.19.0.3
  |  |--172.19.0.4
  |--@ungrouped:
```

I'm not sure if this change would break in other cases
Could you kindly review this when you have time?